package.json

```diff
@@ -2,14 +2,11 @@
   "name": "IV_HorseMatch",
   "version": "1.0.0",
   "description": "Proyecto de IV",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "keywords": [],
   "author": "Marta Huete Martínez",
   "license": "ISC",
-  "dependencies": {
-    "@pnpm/lockfile-file": "^6.0.2"
+  "devDependencies": {
+    "jest": "^29.3.1",
+    "make": "^4.2.1"
```

Comment:
¿Estás segura que este es el make del que habla todo el mundo? (Tercera vez que lo pregunto)

Reply:
No se a que se refiere. Es la versión de make que tengo yo en mi repositorio.

Reply:
Si no lo entendías, ¿por qué lo borras simplemente?

Vamos a plantearlo de otra forma. ¿El `make` que disteis en FS y habéis usado en muchas otras asignaturas es un módulo en javascript que por tanto se instala de esta forma?

Dicho de otro modo, os rogaría que si no sabéis realmente qué herramienta estáis usando, no la elijáis como *task runner* en el objetivo 3.

Reply:
Vale, creo que ya lo entendí. Entonces según entiendo no debería de haber creado esa dependecia. Por ello procedo a borrarla.